database/seeders/AdminSeeder.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\User;
+use Illuminate\Database\Seeder;
+
+class AdminSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        User::where('username', 'shakyaprabeg')
+            ->update(['is_admin' => true]);
```

Comment:
how about adding more seeders using factory maybe as AdminUserFactory and its corresponding Seeder class. I totally got that before u had a user named shakyaprabeg and now yr simply updating it. Try seeding it with factory as well